Packs/PingCastle/pack_metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "PingCastle",
+  "description": "Windows-based utility to audit the risk level of your AD infrastructure and check for vulnerable practices",
```

Comment:
-> "Active directory is quickly becoming a critical failure point in any big sized company, as it is both complex and costly to secure. PingCastle is a Windows-based utility to audit the risk level of your AD infrastructure and check for vulnerable practices. The Integrations and Playbooks in this allows you to listen for PingCastle reports, create an incident based on that report, upload the XML Report to the War Room as a file, and so on"